pkg/test/store/postgres/test_container.go

```diff
@@ -153,3 +157,46 @@ func (v *PostgresContainer) Config(dbOpts ...DbOption) (*pg_config.PostgresStore
 	GinkgoLogr.Info("Database successfully created and started", "name", cfg.DbName)
 	return cfg, nil
 }
+
+func (v *PostgresContainer) PrintDebugInfo(exportDbName string, expectedDbPort int) {
+	logEntries := make([]string, 0)
+	container := v.container
+	state, _ := container.State(context.TODO())
+	logEntries = append(logEntries, "db container details:")
+	logEntries = append(logEntries, fmt.Sprintf("trying to connect db %s on port %d, latest container: %s",
+		exportDbName, expectedDbPort, container.GetContainerID()))
+
+	if state != nil {
```

Comment:
```suggestion
	if state, _ := container.State(context.TODO()); state != nil {
```

generally just limiting the scope of variables as much as possible, couple more spots in the function.

Reply:
Accepted!